### ISSUE TITLE
nvm not found

### ISSUE THREAD

**Issue by nicolo-ribaudo** [railsware/upterm#1206]
When I run `nvm --version`, upterm (0.4.1) throws `Upterm: command "nvm" not found.`. It works in the default Ubuntu terminal.

**Comment by halink0803** [comment]
I think this question has been raised before in #975, you can use [n](https://github.com/tj/n) as an alternative.

**Comment by nicolo-ribaudo** [comment]
Thank you. I'keeping this open (unless the maintainers want to close it) because, even if there is an alternative, `nvm` is quite popular and could be nice to be able to use it.

**Comment by halink0803** [comment]
Yes, I am more concerned about **virtualenv** issue but it seems having the same problem. If they can solve **virtualenv** issue then nvm problem can resolve also.

**Comment by muescha** [comment]
any new updates how to use `nvm`?
